examples/android/android.bazelproject

```diff
@@ -0,0 +1,11 @@
+directories:
+  .
+
+targets:
+  //:android
```

Comment:
this seems to be `//examples/android:android`

Reply:
The bazelproject is relative to the example workspace.